script/karma

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+task="$1"
+
+if [ "$task" = "run" ] || [ "$task" = "start" ]; then
+  exec bundle exec rake "karma:$task" RAILS_ENV=test
+else
+  echo "Usage:"
+  echo "  $0 run  # to run the tests once"
```

Comment:
If you add another space in front of the hash `#`, the comments should align in the same column.